amr-wind/wind_energy/actuator/disk/constant_ct_ops.H

```diff
@@ -0,0 +1,185 @@
+#ifndef CONSTANT_CT_OPS_H_
+#define CONSTANT_CT_OPS_H_
+
+#include "amr-wind/wind_energy/actuator/disk/ConstantCt.H"
+#include "amr-wind/wind_energy/actuator/actuator_ops.H"
+#include "amr-wind/utilities/tensor_ops.H"
+
+namespace amr_wind {
+namespace actuator {
+namespace ops {
+
+template <>
+struct ReadInputsOp<ConstantCt, ActSrcDisk>
+{
+    void operator()(ConstantCt::DataType& data, const utils::ActParser& pp)
+    {
+        auto& ddata = data.meta();
+        auto& info = data.info();
+        pp.get("num_force_points", ddata.num_force_pts);
+        // eventually we will change velocity sampling location
+        // pp.get("num_vel_points_r", ddata.num_vel_points_r);
+        // pp.get("num_vel_points_t", ddata.num_vel_points_t);
+        // pp.get("diameters_to_sample_velocity", ddata.diameters_to_sample);
+        pp.get("disk_center", ddata.center);
```

Comment:
My apologies, I should have mentioned this earlier.  Is it easy to make a change to use `base_position` and `hub_height` instead, so it follows [turbine_utils.cpp](https://github.com/Exawind/amr-wind/blob/6af41101c73b89b1c89053ffab7ef09efcf37ac7/amr-wind/wind_energy/actuator/turbine/turbine_utils.cpp#L29-L31)?  (`disk_center` will then be `base_position` + `[hub_height, 0.0, 0.0]`).  `disk_center` can still be an optional input, but my thought is to standardize the inputs so that it's easier to switch turbine models with the fewest changes possible.

Reply:
yeah that's not a problem.